include/exadg/incompressible_navier_stokes/user_interface/parameters.cpp

```diff
@@ -791,6 +853,9 @@ Parameters::print_parameters_temporal_discretization(dealii::ConditionalOStream
   print_parameter(pcout, "Temporal discretization method", temporal_discretization);
   print_parameter(pcout, "Treatment of convective term", treatment_of_convective_term);
 
+  if(turbulence_model_data.is_active || generalized_newtonian_model_data.is_active)
```

Comment:
we have now a member function for this or-combination of parameters :), which should be called here

Reply:
:eyes: